content/general/concepts/nft/nft.md

```diff
@@ -0,0 +1,29 @@
+---
+Title: 'NFT'
+Description: 'Non-fungible token which act like a digital representation to prove a ownership for a digital good backed by blockchain technology.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Development'
```

Comment:
Is "Web Development" the best subject for this?  "Cybersecurity" would seem more apt when dealing with the subject of blockchains etc.

Reply:
Yeah I think that's fine for now. Because we don't have another subject for web3.